beetsplug/subsonicplaylist.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2019, Joris Jensen
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+from __future__ import absolute_import, division, print_function
+
+from hashlib import md5
+import xml.etree.ElementTree as ET
+import random
+import string
+import requests
+from beets import plugins
+
+from beets.ui import Subcommand
+
+from beets.plugins import BeetsPlugin
+
+__author__ = 'https://github.com/MrNuggelz'
+
+
+class SubsonicPlaylistPlugin(BeetsPlugin):
+
+    def __init__(self):
+        super(SubsonicPlaylistPlugin, self).__init__()
+        self.config.add(
+            {
+                'delete': False,
+                'playlist_ids': [],
+                'playlist_names': [],
+                'username': '',
+                'password': ''
+            }
+        )
+        self.config['password'].redact = True
+
+    def update_tags(self, playlist_dict, lib):
+        for query, playlist_tag in playlist_dict.items():
+            query = 'artist:"{}" album:"{}" title:"{}"'.format(*query)
+            items = lib.items(query)
+            if len(items) <= 0:
+                self._log.warn(u"{} | track not found ({})", playlist_tag,
+                               query)
+                continue
+            for item in items:
+                item.update({'subsonic_playlist': playlist_tag})
```

Comment:
This can be written more simply as:
```suggestion
                item.sbsonic_playlist = playlist_tag
```